src/Commands/AbstractDrupalCommands.php

```diff
@@ -316,8 +316,57 @@ public function drushSetup(array $options = [
     }
 
     /**
-     * Setup Drupal settings overrides.
+     * Setup Drupal services file.
+     *
+     * This command will copy the content of the file given in service-parameters to "services.yml" file in
+     *  the site directory.
+     *
+     * The services file name can be set in the Task Runner
+     * configuration by setting the "drupal.service_parameters" property.
+     *
+     * @command drupal:services-setup
+     *
+     * @option service-parameters       Drupal services filename.
+     * @option root                     Drupal root.
+     * @option sites-subdir             Drupal site subdirectory.
+     * @option force                    Drupal force generation of a new settings.php.
+     * @option skip-permissions-setup   Drupal skip permissions setup.
      *
+     * @param array $options
+     *
+     * @return \Robo\Collection\CollectionBuilder
+     */
+    public function servicesSetup(array $options = [
+        'service-parameters' => InputOption::VALUE_REQUIRED,
+        'root' => InputOption::VALUE_REQUIRED,
+        'sites-subdir' => InputOption::VALUE_REQUIRED,
+        'force' => false,
+        'skip-permissions-setup' => false,
+    ])
+    {
+        $services_runner_file = $this->getConfig()->get('drupal.service_parameters');
+        $services_options_file = $options['service-parameters'];
+        $services_source_file = empty($services_options_file)? $services_runner_file : $services_options_file;
+
+        if (!file_exists($services_source_file)) {
+            throw new \Exception(sprintf('Services file %s not found.', $services_source_file));
+        }
+
+        $services_destination_file = $options['root'] . '/sites/' . $options['sites-subdir'] . '/services.yml';
+
+        $collection = [];
+        $collection[] = $this->taskFilesystemStack()->copy($services_options_file, $services_destination_file, (bool) $options['force']);
```

Comment:
Here, $services_options_file should be $services_source_file, it's not?

Reply:
True, fixed.